source/gloperate/include/gloperate/painter/AbstractOutputCapability.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+namespace gloperate
+{
+
+template <typename T>
+gloperate::Data<T> * AbstractOutputCapability::getOutput(const std::string & name) const
+{
+    for (AbstractData* output : findOutputs(name))
+    {
+        Data<T>* data = dynamic_cast<Data<T>*>(output);
+        if (data)
+        {
+            return data;
+        }
+    }
+
+    return nullptr;
+}
+
+template <typename T>
+gloperate::Data<T> * AbstractOutputCapability::getOutput() const
+{
+    return dynamic_cast<Data<T>*>(collection::detect(allOutputs(), [](AbstractData * data) { return dynamic_cast<Data<T>*>(data) != nullptr; }, nullptr));
```

Comment:
Where is the include for the ```collection``` helper functions?
It seems like this is currently working coincidentally.